src/tcp.c

```diff
@@ -213,7 +214,8 @@ static int tcp_fsm(struct tcp_conn_tcb *conn, struct tcp_hdr *rs)
 
             rs->tcp_flags |= TCP_ACK;
             rs->tcp_ack_num = rs->tcp_seqno + 1;
-            rs->tcp_seqno = 0; /* TODO Randomize */
+            srand(time(NULL));
```

Comment:
Don't call `srand()` function here since we don't intent to change the seed of `rand()` function. `nstack` might work as a library, allowing the linkage of network programs. Alternatively, you can introduce [fast generation of non-uniform random numbers](https://oroboro.com/non-uniform-random-numbers/), that does not depend on `rand()`.

Reply:
Could I use system call `getrandom() `?, Because it is easy to implement and may have more readability.

Reply:
`getrandom` is Linux-specific and introduced in version 3.17 of the Linux kernel. You can use conditional build like this: https://github.com/neomutt/neomutt/commit/5831e487cc45efa8e43892bf051f7f13161ae03a

`nstack` is designed for portability, and you shall provide another path while `getrandom()` is not available on certain platforms.